patch_api/api/estimates_api.py

```diff
@@ -1563,13 +1550,8 @@ def create_sea_shipping_estimate_with_http_info(
         all_params.append("average_daily_balance_btc_sats")
         all_params.append("average_daily_balance_eth_gwei")
         all_params.append("timestamp")
-        all_params.append("destination_country_code")
-        all_params.append("destination_locode")
-        all_params.append("destination_postal_code")
```

Comment:
@holtbp is it possible that you have uncommitted changes to client-code-generation?